src/pages/Form.tsx

```diff
@@ -0,0 +1,61 @@
+import style from "./Form.module.css";
+import ButtonForm from "../UI/Button";
+import { Fragment } from "react";
+import Personal from "../components/Personal";
+import TravelInfo from "../components/TravelInfo";
+import TripInfo from "../components/TripInfo";
+
+function Form() {
+  let personalData = {};
```

Comment:
Why not keep all these variables in state?

Reply:
I fixed that.